docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst

```diff
@@ -60,7 +60,7 @@ The one marked with an asterisk is the "default" profile, meaning that any ``ver
 .. note::
 
     The output you get may differ.
-    The ``generic`` profile is pre-configured on the virtual machine built for the tutorial (but we are not going to use it here).
+    The ``generic`` profile is pre-configured on the virtual machine and built only for the first part of the tutorial.
```

Comment:
Is this true? Huh.

Reply:
You mean that the *generic* profile is no longer used later? It seems so, yes. It actually creates a few problems IMO.